.github/workflows/ios_deploy_to_testflight_staging.yml

```
@@ -0,0 +1,71 @@
+name: ios-deploy-to-testflight-staging
+on:
+  # Trigger the workflow on push action
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build_and_upload_to_testflight:
+    name: Build And Upload iOS Application To TestFlight
+    runs-on: macOS-latest
+    env:
+      TEAM_ID: ${{ secrets.IOS_TEAM_ID }}
+      APP_STORE_CONNECT_API_KEY_KEY_ID: ${{ secrets.APP_STORE_CONNECT_API_KEY_KEY_ID }}
+      APP_STORE_CONNECT_API_KEY_IS_KEY_CONTENT_BASE64: ${{ secrets.APP_STORE_CONNECT_API_KEY_IS_KEY_CONTENT_BASE64 }}
+      APP_STORE_CONNECT_API_KEY_ISSUER_ID: ${{ secrets.APP_STORE_CONNECT_API_KEY_ISSUER_ID }}
+      SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+      MATCH_REPO: ${{ secrets.MATCH_REPO }}
+      MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install SSH key
+      uses: webfactory/ssh-agent@v0.4.1
+      with:
+        ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+
+    - name: Setup Flutter environment
+      uses: subosito/flutter-action@v2
+      with:
+        channel: 'stable'
+        flutter-version: '3.3.10'
+
+    - name: Create Env file
+      run: |
+        touch .env.staging
+        echo "SECRET=\"MY_SECRET\"" >> .env.staging
+        echo "REST_API_ENDPOINT=\"MY_ENDPOINT\"" >> .env.staging
```

I think it should be updated to this, right? (Please refer to `android_deploy_staging.yml`) 🙏 

```suggestion
      - name: Setup .env.staging
        env:
          ENV_STAGING: ${{ secrets.ENV_STAGING }}
        run: |
          echo "$ENV_STAGING" > .env.staging
```